toptal/picasso
# chore: improve the look of PoweredByGoogle label of the Autocomplete

[TACO-1495](https://toptal-core.atlassian.net/browse/TACO-1495)

### Description

Couple of minor improvements to the `PoweredByGoogle` label of the `Autocomplete` component.

- Replace the current image with a high-dpi version. It is still the same image and we go it from their guidelines
- Put the label into the `fixedFooter` of the `SelectOptions`, so it is more or less inline with the limit option of the [`Select`](https://picasso.toptal.net/?path=/story/forms-select--select#limit) component

Those changes has been discussed with Dia Ibram from the design team 🙏 
https://docs.google.com/document/d/1CTj-QzTSOnLyBOCM_s2goS0PAps0dGu89JfpelCAmro/edit?disco=AAAAZ6nJAbU

### How to test

- Spec should be green ✅
- Open the temploy and check the `Autocomplete` component with `Powered by Google` label on -> https://picasso.toptal.net/power-by-google-hdpi/?path=/story/forms-autocomplete--autocomplete#powered-by-google-label
- It should look fabulous ✨

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="340" alt="Shot 2022-05-25 at 13 18 58" src="https://user-images.githubusercontent.com/1788571/170240884-71dc5801-3a54-4c67-b86d-b30d80a75cb2.png"> | <img width="340" alt="Shot 2022-05-25 at 13 18 42" src="https://user-images.githubusercontent.com/1788571/170240892-9e806c99-d28d-4f2c-ab7f-65d5d937a6b6.png"> |





### Development checks


**Breaking change**


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
